openmmtools/tests/test_integrators.py

```diff
@@ -186,6 +187,27 @@ def test_integrator_decorators():
     assert integrator.acceptance_rate == 1.0
 
 
+def test_single_force_update():
+    """
+    Ensures that addUpdateContextState() is called only once for all custom integrators, except the
+    NonequilibriumLangevinIntegrator which requires an alchmical system.
+    """
+    custom_integrators = get_all_custom_integrators()
+    for integrator_name, integrator_class in custom_integrators:
+        # The NonequilibriumLangevinIntegrator requires an alchemical function.
+        if issubclass(integrator_class, integrators.NonequilibriumLangevinIntegrator):
```

Comment:
There's no reason not to test all integrators, since all should have exactly one instance of `addUpdateContextState()`. I'll change this before I merge.